docs/release_notes/v1.9.5.md

```diff
@@ -0,0 +1,19 @@
+# Dapr 1.9.1
+
+### Fixes cosmosDB etag crash on transaction request
+
+#### Problem
+
+When setting any etag in a transaction request inside CosmosDB, dapr exits due to a nil pointer dereference.
+
+### Impact
+
+This issue impacts all users of CosmosDB state store.
+
+#### Root cause
+
+When setting an eTag on a transaction request in cosmosDB state store, a nil pointer was being dereferenced. 
+
+#### Solution
+
+We have changed the cosmosDB statestore code to correctly use the address rather than the pointer of the eTag.
```

Comment:
Your fix actually created a new struct instance and then got the address of that